core/templates/pages/android-page/android-page.component.html

```diff
@@ -1,175 +1,191 @@
-<div class="oppia-android-page-container">
+<div class="oppia-android-page-container overflow-hidden">
   <div class="oppia-android-page">
-    <div class="oppia-android-page-hero-container">
-      <div class="oppia-android-main-text-container">
-        <div class="oppia-android-main-body">
-          <div class="oppia-android-main-text oppia-android-text-color">
-            <img class="oppia-android-logo" src="/assets/images/logo/48x48_logo_mint.svg" alt="">
-            <span>ppia’s Android App—<br>
-              &nbsp;&nbsp;{{ 'I18N_ANDROID_PAGE_AVAILABLE_FOR_DOWNLOAD_TEXT' | translate }}
-            </span>
-          </div>
-          <div class="oppia-android-beta-description-container">
-            <p class="oppia-android-beta-description"
-               [innerHTML]="'I18N_ANDROID_PAGE_BETA_DESCRIPTION' | translate">
-            </p>
-            <p class="oppia-android-support-text"
-               [innerHTML]="'I18N_ANDROID_PAGE_VERSIONS_SUPPORTED_TEXT' | translate">
-            </p>
-            <a [href]="ANDROID_APP_URL" target="_blank" rel="noopener">
-              <img class="oppia-android-google-play-logo"
-                   src="/assets/images/android/android-landing-google-play.svg"
-                   alt="">
-            </a>
+    <div class="container pl-0">
+      <div class="row">
+        <div class="col-sm-6 col-8">
+          <div class="oppia-android-main-body">
+            <div class="oppia-android-main-text oppia-android-text-color">
+              <div>
+                {{ 'I18N_ANDROID_PAGE_AVAILABLE_FOR_DOWNLOAD_TEXT' | translate }}
+              </div>
+            </div>
+            <div class="oppia-android-beta-description-container">
+              <p class="oppia-android-beta-description"
+                 [innerHTML]="'I18N_ANDROID_PAGE_BETA_DESCRIPTION' | translate">
+              </p>
+              <p class="oppia-mobile-only oppia-android-support-text">
+                We support all Android versions going back to Lollipop (Android 5).
+              </p>
+              <a [href]="ANDROID_APP_URL" target="_blank" rel="noopener">
+                <img class="oppia-android-google-play-logo"
+                     src="/assets/images/android/android-landing-google-play.svg"
+                     alt="">
+              </a>
+            </div>
           </div>
         </div>
-      </div>
-      <div class="oppia-android-hero-image-container">
-        <img class="oppia-android-hero-image" src="/assets/images/android/android-landing-0.svg" alt="">
+        <div class="col-sm-6 col-4">
+          <img class="oppia-android-hero-image" src="/assets/images/android/android-landing-0.svg" alt="">
```

Comment:
Missing alt text. All alt text must be filled in with a sentence that makes sense for someone who cannot see the image on the page -- please check your PR.

Reply:
Done.

Though for this particular case, a blank alt text should be OK, since the Hero image can be considered as a decorative image IMO. Per (https://www.w3.org/WAI/tutorials/images/decorative/) the recommendation for such images is to leave the alt text empty.